packages/sdk/src/libs/stores/google-datastore/store.ts

```diff
@@ -1,8 +1,8 @@
-import type { Store } from "..";
+import type { SortedStore } from "..";
```

Comment:
I've not seen this kind of import syntax before. the `from ".."`, what does it mean exactly?